components/JoinCommunityButton/JoinCommunityButton.tsx

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 
 import { CustomLink } from "../CustomLink";
-import { StyledJoinCommunityButton } from "./JoinCommunityButton.styled";
+import {
+  StyledCustomLink,
+  StyledJoinCommunityButton,
+} from "./JoinCommunityButton.styled";
 
 export const JoinCommunityButton = () => {
```

Comment:
can't we just have a more generic component like Button & ButtonOutline? that way you would just pass the children, in this case "Join The community" in some span and we can reuse it. Or would you like to create component for each button on the page? 😁

Reply:
Changed to Button